misc/syntax/visualstudio/rego/language-configuration.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I will replace this with https://github.com/tsandall/vscode-rego/blob/master/language-configuration.json after merging unless you tell me not to.